wordpress-mobile/WordPress-iOS
# Revisions: Add History button and basic table view

Refs #10304

Adds the button to view the new revisions UI, and a basic table view to build on.

![screen recording 2018-10-31 at 11 16 04 pm](https://user-images.githubusercontent.com/517257/47835916-6c1ada00-dd63-11e8-818a-5f200e8c58ea.gif)

To test:
- open a post with some revisions, ensure the history button appears and shows a list of revisions
- open a post without revisions, ensure the history button doesn't appear
